src/sentry/api/endpoints/assistant.py

```diff
@@ -33,11 +35,14 @@ class AssistantEndpoint(Endpoint):
 
     def get(self, request):
         """Return all the guides the user has not viewed or dismissed."""
-        exclude_ids = set(AssistantActivity.objects.filter(
+        guides = deepcopy(GUIDES)
+        seen_ids = set(AssistantActivity.objects.filter(
             user=request.user,
         ).values_list('guide_id', flat=True))
-        result = {k: v for k, v in GUIDES.items() if v['id'] not in exclude_ids}
-
+        result = {}
+        for k, v in guides.items():
+            v['seen'] = k in seen_ids
```

Comment:
Hmm I think this should be `v['id'] not in seen_ids`

Reply:
it's `v[k]['id']` but good catch.

Reply:
Whoops, nope you're right.